README.md

```diff
@@ -1 +1 @@
-# Code-Review
\ No newline at end of file
+# Personal-Portfolio
```

Comment:
**README.md** file and commits need to be professional. 😊 Make sure you add sources, authors, a title, and a description to your project. That file is the first thing someone sees first when checking your repo. ✅ 
As well, **commits** should be meaningful, short and straight to the point.